### DISCUSSION TITLE
Run Command on Vercel

### DISCUSSION THREAD

**Discussion by Hanad-Muqaddar** [vercel/community#4131] [Help]
### Summary

I have deployed a python application on Vercel, My appliaction need google chrome, because one functionality requires selenium to use chrome. But it doesn't installed on server. I got an error that google chrome command not found. So how can i fix this?? Please help me out with that.

### Example

https://journaya-app.vercel.app/

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/55221009/7b80d2b0-51f3-4ca7-90fe-57dca9b9d586)
This is the error i found in logs.

**Comment by zheng-liu-seattle** [comment]
It has 50mb limiation, using chrome is impossible, but you can use remote chrome, like browseless
